feat: [CDS-91176]: add doc for delegate hotfix 82306 (#5442)
harness/developer-hub: release-notes/delegate.md

## February 2024

> 

Reviewer: @lovish1999 Is this HF 82305 (in the title) or 82306 (here in the markdown)?

Author: @brian-f-harness , it is 82306. There are some issues with the 82305 build. I will mark the PR ready for review in some time. Currently, have changed it to draft.

> ### Version 24.02.82306 <!--  February 16, 2024 -->

Reviewer: ```suggestion
### Version 24.02.82305 <!--  February 16, 2024 -->
```

#### Hotfix

> - Added default values for minimum healthy percentage as 90 and maximum healthy percentage as 110 for instance refresh operation which is performed during ASG Rolling deployments to prevent downtime of services. (CDS-91335, ZD-57686)

Reviewer: ```suggestion
- Added default values for minimum healthy percentage as 90 and maximum healthy percentage as 110 for the instance refresh operation that is performed during ASG Rolling deployments to prevent service downtime. (CDS-91335, ZD-57686)
```
